<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AGENDA</class>
 <widget class="QDialog" name="AGENDA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agenda</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15,15, 15);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_voltar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>ATENÇÃO: Qualquer dúvida no uso do programa contate o</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>desenvolvedor!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_confirma">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>confirmar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tituloAntigo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>Título:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Informações ANTIGAS:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TipoAntigo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(103, 103, 103);</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prova</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dever de casa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Atividades extras</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Informações NOVAS:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TipoNovo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(103, 103, 103);</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prova</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dever de casa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Atividades extras</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tituloNovo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>Título:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_descrio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(94,141,235);</string>
   </property>
   <property name="text">
    <string>Descrção:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AGENDA</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AGENDA</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
